taxonomies/categories.txt

```diff
@@ -47620,6 +52358,13 @@ nl_be:Aardappelen
 se:Potatisar, potatis
 pnns_group_2:en:Potatoes
 
+<en:British vegetables
```

Comment:
```suggestion
```

Reply:
I've removed the British veg alltogether.